android/src/main/java/com/bitmovin/player/reactnative/RNPlayerView.kt

```diff
@@ -203,6 +204,13 @@ class RNPlayerView(
         }
     }
 
+    /**
+     * Make sure this is called after [addPlayerView], otherwise the subtitleView will be overdrawn by the player!
+     */
```

Comment:
Is that comment true? You are adding the `playerView` on index 0, so there shouldn't be any player in front of the `SubtitleView`. Do I miss something?

Reply:
removed: https://github.com/bitmovin/bitmovin-player-react-native/pull/316/commits/5f185a9687dd49771a27cf81d0a3089c4f5f05ab

add different comment: https://github.com/bitmovin/bitmovin-player-react-native/pull/316/commits/054bc2786ca5588a10bee9d7981bfbbc43336d5a